src/local/butler/reproduce.py

```diff
@@ -272,7 +279,15 @@ def _reproduce_crash(testcase_id, build_directory):
 
 def execute(args):
   """Attempt to reproduce a crash then report on the result."""
-  result = _reproduce_crash(args.testcase, args.build_dir)
+  try:
+    result = _reproduce_crash(args.testcase, args.build_dir)
+  except ReproduceToolException as exception:
+    print(exception)
+    return
+
+  if result.is_crash():
+    print('Test case reproduced successfully:\n')
```

Comment:
Remove :\n from this and Output:\n from next

just do a common
print('Output:\n' + result.get_stacktrace())

Reply:
Done